src/lib/PostgresMeta.ts

```diff
@@ -0,0 +1,43 @@
+import { ClientConfig } from 'pg'
+import PostgresMetaColumn from './PostgresMetaColumn'
+import PostgresMetaConfig from './PostgresMetaConfig'
+import PostgresMetaExtension from './PostgresMetaExtension'
+import PostgresMetaFunction from './PostgresMetaFunction'
+import PostgresMetaPolicy from './PostgresMetaPolicy'
+import PostgresMetaPublication from './PostgresMetaPublication'
+import PostgresMetaRole from './PostgresMetaRole'
+import PostgresMetaSchema from './PostgresMetaSchema'
+import PostgresMetaTable from './PostgresMetaTable'
+import PostgresMetaType from './PostgresMetaType'
+import PostgresMetaVersion from './PostgresMetaVersion'
+import { init } from './db'
+
+export default class PostgresMeta {
+  query: Function
```

Comment:
@soedirgo I see `Function` type used for `query` in a couple of places so I wanted to suggest an alternative.

Perhaps something like this is more descriptive, we can use it in all the other places `query` comes up, and help us down the line as we continue development:

```
type queryAsyncFunc = {
  (sql: string): Promise<{ data: any[] | null, error: { message: string } | null}>;
}

query: queryAsyncFunc
```

Reply:
Yeah, the typings aren't final (there are still some `any`s), `Function` was just for convenience. But I'll update the type for `query`.

Reply:
@soedirgo oops, spoke too soon!